sdk/communication/azure-communication-chat/azure/communication/chat/_generated/models/_models_py3.py

```diff
@@ -207,758 +310,876 @@ def __init__(
 
 
-class SendChatMessageRequest(msrest.serialization.Model):
+class SendChatMessageRequest(_serialization.Model):
     """Details of the message to send.
 
     All required parameters must be populated in order to send to Azure.
 
-    :param content: Required. Chat message content.
-    :type content: str
-    :param sender_display_name: The display name of the chat message sender. This property is used
+    :ivar content: Chat message content. Required.
+    :vartype content: str
+    :ivar sender_display_name: The display name of the chat message sender. This property is used
      to populate sender name for push notifications.
-    :type sender_display_name: str
-    :param type: The chat message type. Possible values include: "text", "html", "topicUpdated",
-     "participantAdded", "participantRemoved".
-    :type type: str or ~azure.communication.chat.models.ChatMessageType
-    :param metadata: Message metadata.
-    :type metadata: dict[str, str]
+    :vartype sender_display_name: str
+    :ivar type: The chat message type. Known values are: "text", "html", "topicUpdated",
+     "participantAdded", and "participantRemoved".
+    :vartype type: str or ~azure.communication.chat.models.ChatMessageType
     """
 
     _validation = {
-        'content': {'required': True},
+        "content": {"required": True},
     }
 
     _attribute_map = {
-        'content': {'key': 'content', 'type': 'str'},
-        'sender_display_name': {'key': 'senderDisplayName', 'type': 'str'},
-        'type': {'key': 'type', 'type': 'str'},
-        'metadata': {'key': 'metadata', 'type': '{str}'},
+        "content": {"key": "content", "type": "str"},
+        "sender_display_name": {"key": "senderDisplayName", "type": "str"},
+        "type": {"key": "type", "type": "str"},
     }
 
     def __init__(
         self,
         *,
         content: str,
         sender_display_name: Optional[str] = None,
-        type: Optional[Union[str, "ChatMessageType"]] = None,
-        metadata: Optional[Dict[str, str]] = None,
-        **kwargs
-    ):
-        super(SendChatMessageRequest, self).__init__(**kwargs)
+        type: Optional[Union[str, "_models.ChatMessageType"]] = None,
+        **kwargs: Any
+    ) -> None:
+        """
+        :keyword content: Chat message content. Required.
+        :paramtype content: str
+        :keyword sender_display_name: The display name of the chat message sender. This property is
+         used to populate sender name for push notifications.
+        :paramtype sender_display_name: str
+        :keyword type: The chat message type. Known values are: "text", "html", "topicUpdated",
+         "participantAdded", and "participantRemoved".
+        :paramtype type: str or ~azure.communication.chat.models.ChatMessageType
+        """
+        super().__init__(**kwargs)
         self.content = content
         self.sender_display_name = sender_display_name
         self.type = type
-        self.metadata = metadata
```

Comment:
Is this a breaking change? Or was this never exposed in the public API?

Reply:
This was removed by mistake, the swagger has been updated and this is reverted when using autorest